pynucastro/nucdata/nucleus.py

```diff
@@ -161,6 +161,30 @@ def from_cache(cls, name, dummy=False):
             cls._cache[key] = Nucleus(name, dummy)
```

Comment:
Not directly related to your changes, but I just noticed this should use `cls`:
```suggestion
            cls._cache[key] = cls(name, dummy)
```